src/include/cluster/dialogs.rb

```diff
@@ -511,10 +513,11 @@ def CommunicationLayout
 
       riface = CheckBoxFrame(
         Id(:enable2),
-        Opt(:notify),
+        Opt(:noAutoEnable, :notify),
```

Comment:
what is this opt? I think it does not exists. What it should do?

Reply:
It is caused by a wired issue of https://bugzilla.suse.com/show_bug.cgi?id=1179007

To short:
Explicit disable the checkbox doesn't work in ncurses UI, but works for others... And only this box.  Petr from L3 team give a possible analyze and a workaround of ":noAutoEnable" to make the parent box **disabled** by default. And it works in the test...